components/modules/Header/Header.tsx

```diff
@@ -1,6 +1,56 @@
+import styles from '../../../styles/header/header.module.scss'
```

Comment:
Styles in order should be below all, and outsourced imports above all

Reply:
done